Tasks/Common/azure-arm-rest/azure-arm-resource.ts

```diff
@@ -12,7 +12,7 @@ export class ResourceManagementClient extends azureServiceClient.ServiceClient {
     constructor(credentials: msRestAzure.ApplicationTokenCredentials, subscriptionId: string, options?: any) {
         super(credentials, subscriptionId);
 
-        this.apiVersion = '2016-07-01';
+        this.apiVersion = '2017-05-10';
```

Comment:
I've updated the version only for RG REST calls, i.e. Deployments, Creating RG etc to avoid impacting the other rest calls. 
I've checked out a few scenarios using this api-version for deployments, they were succeeding